OneCommunityGlobal/HighestGoodNetworkApp
# Sriram: Added tests for WBS component

# Description

Unit testing for the `src/components/Projects/WBS/wbs.test.jsx`

## Related PRS (if any):
None
…

## Main changes explained:
- Deleted the existing tests because they are testing WBSItem component instead of WBS.
- Added tests for WBS component

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. run `npm test src/components/Projects/WBS/wbs.test.jsx`
4. Also try running all test suites `npm test`
